src/commands/edge/drivers/package.ts

```diff
@@ -196,21 +212,18 @@ export default class PackageCommand extends EdgeCommand {
 
 			this.processProfiles(projectDirectory, parsedConfig, zip)
 
-			if (flags['dry-run']) {
+			if (flags['build-only']) {
 				zip.generateNodeStream({ type: 'nodebuffer', streamFiles: true, compression: 'DEFLATE' })
-					.pipe(fs.createWriteStream('edge.zip'))
+					.pipe(fs.createWriteStream(flags['build-only']))
 					.on('finish', () => {
-						this.log('wrote edge.zip')
+						this.log(`wrote ${flags['build-only']}`)
 					})
-				// This is a little hacky for now...we're making a fake driver
-				// to return for dry run mode. We should probably support for
-				// an alternate return type to be used in dry-run mode.
-				return { driverId: 'dry-run-mode', name: 'n/a', packageKey: 'n/a', version: 'n/a' } as EdgeDriver
```

Comment:
`EdgeDriver` import can be removed above.

Reply:
removed